tests/resolvers/ChatMessage/replyTo.spec.ts

```diff
@@ -0,0 +1,95 @@
+import "dotenv/config";
+import { replyTo as replyToResolver } from "../../../src/resolvers/ChatMessage/replyTo";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { ChatMessage } from "../../../src/models";
+import { beforeAll, afterAll, describe, it, expect, vi } from "vitest";
+import type { TestChatMessageType } from "../../helpers/chat";
+import {
+  createTestChatMessageWithoutReply,
+  createTestChatMessage,
+} from "../../helpers/chat";
+import { Types } from "mongoose";
+import { MESSAGE_NOT_FOUND_ERROR } from "../../../src/constants";
+
+let testChatMessage: TestChatMessageType;
+let testChatMessageWithoutReply: TestChatMessageType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  const temp = await createTestChatMessageWithoutReply();
+  const temp1 = await createTestChatMessage();
+  testChatMessageWithoutReply = temp[3];
+  testChatMessage = temp1[3];
+});
+
+afterAll(async () => {
```

Comment:
If `connect()` or data creation fails in `beforeAll`, `afterAll` could still try to run `disconnect()`, which might throw errors since `MONGOOSE_INSTANCE` could be undefined. Add a conditional check before calling disconnect.

**Example**
```javascript
afterAll(async () => {
  if (MONGOOSE_INSTANCE) {
    await disconnect(MONGOOSE_INSTANCE);
  }
});
```